tests/rptest/services/admin.py

```diff
@@ -20,9 +24,31 @@ class Admin:
     value is a decoded dict of the JSON payload, for other requests
     the successful HTTP response object is returned.
     """
-    def __init__(self, redpanda):
+    def __init__(self, redpanda, default_node=None, retry_codes=None):
         self.redpanda = redpanda
 
+        self._session = requests.Session()
+
+        self._default_node = default_node
+
+        # - We retry on 503s because at any time a POST to a leader-redirected
+        # request will return 503 if the partition is leaderless -- this is common
+        # at the very start of a test when working with the controller partition to
+        # e.g. create users.
```

Comment:
I think this addresses my comment a few commits back in the series about needing a retry when there is no known leader and thus we can't rely on a redirect.